pysb/generator/kappa.py

```diff
@@ -15,24 +18,31 @@ def get_content(self):
 
     def generate_content(self):
         self.__content = ''
-        #self.generate_parameters()
         #self.generate_compartments()
 
         # Agent declarations appear to be required in kasim
         # but prohibited in complx
         if (self.dialect == 'kasim'):
             self.generate_molecule_types() 
+            # Parameters, variables, and expressions are allowed in kasim
+            self.generate_parameters()
 
         self.generate_reaction_rules()
         self.generate_observables()
         self.generate_species()
 
-    #def generate_parameters(self):
-    #    self.__content += "begin parameters\n"
-    #    max_length = max(len(p.name) for p in self.model.parameters)
-    #    for p in self.model.parameters:
-    #        self.__content += ("  %-" + str(max_length) + "s   %e\n") % (p.name, p.value)
-    #    self.__content += "end parameters\n\n"
+    def generate_parameters(self):
+        names = [x.name for x in self.model.parameters] + \
+                [x.name for x in self.model.expressions]
+        for p in self.model.parameters:
+            self.__content += ("%%var: \'%s\' %e\n") % (p.name, p.value)
```

Comment:
Single quotes don't need to be escaped within a double-quoted string. There is a lot of that in this PR... Could probably grep for all the instances and fix them up pretty quickly.